tests/unit/business-lookup-services.spec.ts

```diff
@@ -44,4 +44,36 @@ describe('Business Lookup Services', () => {
 
     sinon.restore()
   })
+
+  it('returns active and historical results when status is empty', async () => {
+    const result1 = {
+      identifier: 'FM1000001',
+      legalType: 'GP',
+      name: 'INDUSTIRES GP',
+      score: 12.966249,
+      status: 'ACTIVE'
+    }
+    const result2 = {
+      bn: '701819922',
+      identifier: 'FM1000002',
+      legalType: 'SP',
+      name: 'KK CONSTRUCTION',
+      score: 10.642771,
+      status: 'HISTORICAL'
+    }
+
+    // mock successsful search
+    sinon.stub(axios, 'get')
+      .withArgs('https://search.api.url/businesses/search/facets?start=0&rows=20&categories=legalType:' +
+        'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,SP,CC,XL,FI,XCP,PA&query=value:FM100000')
+      .returns(new Promise(resolve => resolve({ data: { searchResults: { results: [result1, result2] } } })))
+
+    // search and look at results
+    const results = await BusinessLookupServices.search('FM100000', '')
```

Comment:
Calling `search()` with empty parameter `status`, which means "match all statuses".

Compare with line 42 above with undefined parameter `status`, which means "match ACTIVE".

Reply:
That is good to know!

Reply:
It's documented in the function's JSDOC header anyway.

I'm just calling out here that I'm testing a couple of different scenarios.